.gitignore

```diff
@@ -1,2 +1,2 @@
 *.pyc
-results
```

Comment:
Why remove `results` from `.gitignore`? Isn't that where most users will direct the results?